src/plugins/telemetry/server/routes/telemetry_usage_stats.ts

```diff
@@ -31,9 +35,18 @@ export function registerTelemetryUsageStatsRoutes(
     async (context, req, res) => {
       const { unencrypted, refreshCache } = req.body;
 
+      const security = getSecurity();
+      if (security && unencrypted) {
+        const { hasAllRequested } = await security.authz
+          .checkPrivilegesWithRequest(req)
+          .globally({ kibana: 'decryptedTelemetry' });
```

Comment:
You need to check the serialized privilege string here. I'd suggest adding a comment to clarify this design decision for future maintainers, too 😄 
```suggestion
        // Normally we would use `options: { tags: ['access:decryptedTelemetry'] }` in the route definition to check authorization for an
        // API action, however, we want to check this conditionally based on the `unencrypted` parameter. In this case we need to use the
        // security API directly to check privileges for this action. Note that the 'decryptedTelemetry' API privilege string is only
        // granted to users that have "Global All" or "Global Read" privileges in Kibana.
        const { checkPrivilegesWithRequest, actions } = security.authz;
        const privileges = { kibana: actions.api.get('decryptedTelemetry') };
        const { hasAllRequested } = await checkPrivilegesWithRequest(req).globally(privileges);
```

Reply:
It works! Thank you! Is there any documentation I could have gone to earlier so I wouldn't bother you with this wrong usage?

Reply:
You're welcome!

Here's where we have an example of securing an HTTP route with an API privilege string via an `access:` tag: https://www.elastic.co/guide/en/kibana/current/development-security.html#example-2-dev-tools

Unfortunately you couldn't use an `access:` tag here.
AFAIK we don't really have any dev docs or TS docs that explain how to use the Security plugin's API to check privileges directly. The only TS docs I could find is this snippet:

https://github.com/elastic/kibana/blob/2c4e795e1807b0a6a681d20da49829b2512400cd/x-pack/plugins/security/server/authorization/actions/actions.ts#L16-L19

That's not too helpful 😅 

I added an issue for us to add docs for the authorization service: #126355